app/src/lib/string.ts

```diff
@@ -0,0 +1,13 @@
+export const getInitials = (name: string) => {
```

Comment:
I think it is probably a good idea to change name type to `string | undefined` and then return the default "XX" if undefined as well, this way we dont have to have a mention of "XX" anywhere else except this component and in the `AccountHeaderDropdown` file you can just do `getInitials(user?.displayName)` because right now we have default "XX" being set in the component and defaults being set in this function

Reply:
done